lib/menu.ts

```diff
@@ -301,12 +318,27 @@ function findAncestorChild(ancestor: Element, elem: Element|null): Element|null
  * Sets the width of the given menu element to match the given container element. Used by
  * IMenuOptions setting 'stretchToSelector'.
  */
-function stretchMenuToContainer(menuElem: HTMLElement, containerElem: Element): void {
-  const style = menuElem.style;
+function stretchMenuToContainer(menuEl: HTMLElement, containerElem: Element): void {
+  const style = menuEl.style;
   style.minWidth = containerElem.getBoundingClientRect().width + 'px';
   style.marginLeft = style.marginRight = '0';
 }
 
+/**
+ * A version of dom.on('mouseover') that doesn't start firing until there is first a 'mousemove'.
+ * This way if an element is created under the mouse cursor (triggered by the keyboard, for
+ * instance) it's not immediately highlighted, but only when a user moves the mouse.
+ */
+function mouseOverOnMove<T extends EventTarget>(callback: EventCB<MouseEvent, T>): DomMethod<T> {
```

Comment:
I'm not sure I understand the problem this is solving. If a submenu is opened with a keyboard and one of the menu items happens to fall under the current mouse cursor it won't be highlighted until you move the mouse?

Reply:
Correct! So does that mean it's explained well enough? Before, the annoyance was that if a menu (or select-like menu) is expected to be opened with a particular item selected, the user might never actually see that item selected if the mouse happened to be over the area where the menu opens. I checked that native menus on MacOS (and select dropdowns) behave like this: only select items in a newly-opened menu once the mouse is moved.

Reply:
That's a nice touch!